### DISCUSSION TITLE
Bounding box need to hug text in fabric js

### DISCUSSION THREAD

**Discussion by Osam2023** [community/community#127459] [Programming Help]
### Select Topic Area

Question

### Body

how bounding box to hug text in fabric js, in canvas when generating text there is a spaces between letter and bounding box, need to remove those gaps, so bounding box hugs text letter edges, see attached image
![image](https://github.com/community/community/assets/171554799/e7d05ee2-2532-4309-b6d1-ca06e11994e9)

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Osam2023 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
